packages/katello/rubygem-qpid_messaging/rubygem-qpid_messaging.spec

```diff
@@ -70,6 +71,9 @@ rm -rf %{buildroot}%{gem_instdir}/ext
 %license %{gem_instdir}/LICENSE
 
 %changelog
+* Fri Jul 13 2018 Eric D. Helms <ericdhelms@gmail.com> - 1.36.0-2
+- rebuilt
```

Comment:
And here :)

Reply:
This was a cherry pick so I'd rather not change it.

Reply:
Fair enough